.github/workflows/ansible-lint-sap_general_preconfigure.yml

```
@@ -24,15 +24,19 @@ jobs:
 
     steps:
       - name: Check out the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@main
```

```suggestion
        uses: actions/checkout@v3
```
should be v3 instead of main because of main is used for upstream

---

Closing for now - we will continue the discussion in https://github.com/sap-linuxlab/community.sap_install/discussions.